symfony/templates/deceasedreports/review.html.twig

```diff
@@ -18,7 +18,8 @@
       {% if report.denialReason %}
       <dt>Denial Reason</dt>
       <dd>{{ report.denialReasonDisplay }} {% if report.denialReason == 'OTHER' %}- {{ report.denialReasonOtherDescription }}{% endif %}</dd>
-      {% endif %}
+
+      {% else %}
```

Comment:
Should we account for `reportStatus` = `cancelled` but no denial reason is set? This should be required, but seems like it would be good to be explicit here. That is, display the denial reason if it is set. And then if the status is not denied, show the information currently in this else block.

Reply:
Not a bad move. I'm _pretty_ sure creating a review in the RDR explicitly requires the reason (fails on creation without it), but that would be an extra safeguard. Fixed in f74c2b51